webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Advanced.js

```diff
@@ -72,11 +68,9 @@ Advanced.propTypes = {
   handleJwtExpiration: PropTypes.func.isRequired,
   handleInvalidField: PropTypes.func.isRequired,
   packages: PropTypes.string,
-  repo: PropTypes.string,
-  repoGpgKeyUrl: PropTypes.string,
+  repoData: PropTypes.array.isRequired,
```

Comment:
Should it be required field? If yes, should it have some default value?

Reply:
I think it should because I want a pre-defined empty state which I can pass down to all the components. If it is a required prop, it does not need a default prop inside the component.